src/Voron/Schema/Updates/From22.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Sparrow;
+using Voron.Impl.FileHeaders;
+using Voron.Impl.Journal;
+
+namespace Voron.Schema.Updates
+{
+    public class From22 : IVoronSchemaUpdate
+    {
+        public unsafe bool Update(int currentVersion, StorageEnvironmentOptions options, HeaderAccessor headerAccessor, out int versionAfterUpgrade)
+        {
+            headerAccessor.Modify(header =>
+            {
+                Memory.Set(header->Journal.Reserved, 0, JournalInfo.NumberOfReservedBytes);
```

Comment:
Replace `NumberOfReservedBytes` with `3`

Reply:
Fixed in https://github.com/ravendb/ravendb/pull/8576